.gitignore

```
@@ -0,0 +1,2 @@
+node_modules/
+package-lock.json
```

```suggestion
package-lock.json

```

Add new line (linting)